### DISCUSSION TITLE
Installing GlueStack UI + Expo + Expo Router (missing Expo Router)

### DISCUSSION THREAD

**Discussion by eybel** [gluestack/gluestack-ui#1670] [Feature Request]
Hi guys.

Amazing project. This is a feature recommendation, not urgent. Its still easy to install Expo Router but...

It would be awesome if you guys offered the installation with Expo but with Router by default or at least optional.

Expo + Expo Router is offered by Expo.

These are the packages that come with expo + expo router:

[EXPO + EXPO ROUTER OFFICIAL INSTALLATION](https://docs.expo.dev/router/installation)

  "dependencies": {
    "expo": "~49.0.15",
    "expo-status-bar": "~1.6.0",
    "react": "18.2.0",
    "react-native": "0.72.6",
    "expo-router": "^2.0.0",
    "react-native-safe-area-context": "4.6.3",
    "react-native-screens": "~3.22.0",
    "expo-linking": "~5.0.2",
    "expo-constants": "~14.4.2",
    "react-native-gesture-handler": "~2.12.0"

**Comment by Viraj-10** [comment]
Hey @eybel, you can use `npm create gluestack@latest` and select Expo router option.
